container_files/assets/www/doc/builtin/functions/EM.md

```diff
@@ -0,0 +1,40 @@
+# EM Function
```

Comment:
Ah, if it's a generic EM training framework then great.  But I'd expect some kind of configuration as to what model it's training.